src/lib-python/ioda_conv_ncio.py

```diff
@@ -286,13 +292,19 @@ def WriteNcMetadata(self, MdataGroup, DimName, Mdata):
                 self._fid.createVariable(NcVname, NcDtype, (DimName))
 
             self._fid[NcVname][:] = Vvals
+            # add units
+            try:
+                self._fid[NcVname].setncattr_string("units", VarUnits[Vname])
+            except KeyError:
+                pass
 
             # If we are writing out the datetime string, then we also
             # need to write out the time offset (from the reference date_time
             # attribute value).
             if (NcVname == "datetime@MetaData"):
                 ToffsetValues = self.ConvertToTimeOffset(Vvals)
                 self._fid[ToffsetName][:] = ToffsetValues
+                # self._fid[ToffsetName].setncattr_string("Units", "hours since "+self._ref_date_time.strftime("%Y-%m-%d %H:%M:%S")+" UTC")
```

Comment:
is this commented intentionally?

Reply:
Just for now so that I didn't have to regenerate the test case data for the marine obs tests (because it would add it to them as well). But I thought it was an important placeholder to have for later (that the time@MetaData means hours since a reference time)